include_core/omrport.h

```diff
@@ -776,7 +776,11 @@ typedef struct J9ProcessorInfos {
 #define OMRPORT_SIG_FLAG_DOES_NOT_MAP_TO_POSIX  0x200000
 #define OMRPORT_SIG_FLAG_SIGHUP  0x400000
 #define OMRPORT_SIG_FLAG_SIGCONT  0x800000
-#define OMRPORT_SIG_FLAG_SIGALLASYNC  (OMRPORT_SIG_FLAG_SIGQUIT | OMRPORT_SIG_FLAG_SIGABRT | OMRPORT_SIG_FLAG_SIGTERM | OMRPORT_SIG_FLAG_SIGRECONFIG | OMRPORT_SIG_FLAG_SIGXFSZ | OMRPORT_SIG_FLAG_SIGINT | OMRPORT_SIG_FLAG_SIGHUP | OMRPORT_SIG_FLAG_SIGCONT)
+#define OMRPORT_SIG_FLAG_SIGWINCH  0x1000000
+#define OMRPORT_SIG_FLAG_SIGALLASYNC \
+			(OMRPORT_SIG_FLAG_SIGQUIT | OMRPORT_SIG_FLAG_SIGABRT | OMRPORT_SIG_FLAG_SIGTERM \
+				| OMRPORT_SIG_FLAG_SIGRECONFIG | OMRPORT_SIG_FLAG_SIGXFSZ | OMRPORT_SIG_FLAG_SIGINT \
```

Comment:
The multiple levels of indentation here aren't my favourite but I won't object if @charliegracie is ok with this.

Reply:
I would have preferred the `|` to be at the end of the line and then each new line lines up with the line above. ie...
```
#define OMRPORT_SIG_FLAG_SIGALLASYNC ( \
			OMRPORT_SIG_FLAG_SIGQUIT | OMRPORT_SIG_FLAG_SIGABRT | OMRPORT_SIG_FLAG_SIGTERM | \
			OMRPORT_SIG_FLAG_SIGRECONFIG | OMRPORT_SIG_FLAG_SIGXFSZ | OMRPORT_SIG_FLAG_SIGINT | \
			OMRPORT_SIG_FLAG_SIGHUP | OMRPORT_SIG_FLAG_SIGCONT | OMRPORT_SIG_FLAG_SIGWINCH)
```